src/main/java/uk/ac/ebi/ega/egacryptor/cryptography/util/Hash.java

```diff
@@ -17,10 +17,10 @@
  */
 package uk.ac.ebi.ega.egacryptor.cryptography.util;
 
+import org.bouncycastle.util.encoders.Hex;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.xml.bind.DatatypeConverter;
```

Comment:
Instead of JAXB, let's use Bouncycastle. No new dependencies were needed to be added.